File: problems/TravelingSalesman/sample/type3/python/main.py
import sys

class TravelingSalesman:
    def solve (self, N, x, y):
        v = []
        for i in range(N):
            v.append(i)
        return v

def main():
    N = int(input())
    x = []
    y = []
    for i in range(N):
        xt,yt = map(int,input().split())
        x.append(xt)
        y.append(yt)
    ts = TravelingSalesman()
    v = ts.solve(N, x, y)
    for i in v:
        print(i)
    sys.stdout.flush()

if __name__ == '__main__':
    main()

File: problems/TravelingSalesman/sample/type3/cpp/main.cpp
#include <iostream>
#include <vector>
using namespace std;

class TravelingSalesman
{
public:
    auto solve (
        const int N, 
        const vector<int> x,
        const vector<int> y)
    {
        vector<int> v(N);
        
        for (int i = 0; i < N; i++) {
            v[i] = i;
        }
         
        return v;
    }
};

int main ()
{
    int N; 
    cin >> N;

    vector<int> x(N), y(N);
    for (int i = 0; i < N; i++) {
        cin >> x[i] >> y[i];
    }

    TravelingSalesman ts;
    vector<int> v = ts.solve(N, x, y);
    for (auto vt: v) {
        cout << vt << endl;
    }

    cout.flush();
    
    return 0;
}
